doc/man3/SSL_CTX_set_ex_data.pod

```diff
@@ -0,0 +1,56 @@
+=pod
+
+=head1 NAME
+
+SSL_CTX_get_ex_data, SSL_CTX_set_ex_data,
+SSL_get_ex_data, SSL_set_ex_data,
+SSL_SESSION_get_ex_data, SSL_SESSION_set_ex_data
+- Store and retrieve extra data from the B<SSL_CTX>, B<SSL> or B<SSL_SESSION>
+
+=head1 SYNOPSIS
+
+ #include <openssl/ssl.h>
+
+ void *SSL_CTX_get_ex_data(const SSL_CTX *s, int idx);
+
+ int SSL_CTX_set_ex_data(SSL_CTX *s, int idx, void *arg);
+
+ void *SSL_get_ex_data(const SSL *s, int idx);
+
+ int SSL_set_ex_data(SSL *s, int idx, void *arg);
+
+ int SSL_SESSION_set_ex_data(SSL_SESSION *s, int idx, void *arg);
+
+ void *SSL_SESSION_get_ex_data(const SSL_SESSION *s, int idx);
+
+=head1 DESCRIPTION
+
+SSL*_set_ex_data() functions can be used to store arbitrary user data into the
+B<SSL_CTX>, B<SSL> or B<SSL_SESSION>. The user must supply a unique index
+which they can subsequently use to retrieve the data using SSL*_get_ex_data().
+
+For more detailed information see L<CRYPTO_get_ex_data(3)> and
+L<CRYPTO_set_ex_data(3)> which implement these functions.
```

Comment:
That's kinda nice, but most importantly, you might want to have people look at `CRYPTO_get_ex_new_index()` for that unique index.